scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -103,7 +103,7 @@ def get_memballoon_type():
     </memballoon>"""
 
     return """    <memballoon model='virtio' autodeflate='on'>
-      <address type='pci' slot='0x05'/>
+      <address type='pci' slot='0x15'/>
```

Comment:
I hadn't noticed that. Good point. I can move the network PCI slot by `0x20`. Would that help?